zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,32 @@
+# Flock webhook
+
+Get Zulip notifications from your Flock channels.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Login into to your [Flock](https://flock.com) account, follow below
+steps given below to get started.
+
+1. Go to **Apps** in the right bottom.
+
+![](/static/images/integrations/flock/001.png)
+
+2. Then, Go to **Admin Panel** -> **Webhooks**.
+
+3. Click on **Add** button next to **Outgoing Webhook**.
+
+![](/static/images/integrations/flock/002.png)
+
+4. Enter the URL you want webhooks to be sent to, in **Callback URL** field.
+
+![](/static/images/integrations/flock/003.png)
+
+5. After filling in other details, click **Save Settings**.
+
+![](/static/images/integrations/flock/004.png)
+
+{!congrats.md!}
+
+![](/static/images/integrations/flock/005.png)
```

Comment:
Some additional comments:
* Set all numbered-steps to begin with `1.`. Markdown will take care of incrementing them for us. :)
* What about the `token` in one of your screenshots? Is that necessary? If so, you should probably mention that!
* I would recommend getting rid of all of the screenshots (except the last one with the sample message). We are trying to move towards a no-screenshots policy unless the third-party UI is so confusing that we absolutely have to include screenshots. :)

Reply:
Oh, that token...that is not necessary for its implementation (it's not even present in JSON responses I got from flock...). Hmm...okay so I'm removing screenshots :)